src/plugins/augeas/README.md

```diff
@@ -8,73 +8,70 @@
 - infos/status = maintained unittest libc configurable
 - infos/description = reading and writing configurations via libaugeas
 
-## INTRODUCTION ##
+## Introduction ##
 
 This is a plugin for reading and writing configuration files with help from Augeas.
 The plugin should be able to read all configuration files for which an Augeas lens exists.
 However, not all stock lenses of Augeas have been tested yet.
 A detailed description of the lens language and a tutorial on how to write new lenses
 can be found at http://augeas.net/
 
-## DEPENDENCIES ##
+## Dependencies ##
 
 - `libaugeas-dev`: You need version 0.16 or higher
 
-## INSTALLATION ##
+## Installation ##
 
 If you have installed Augeas manually, it may be necessary to update the ld configuration. This is especially
 true if an older version of Augeas is installed also. Such a situation may lead to an error similar to this:
 
-/usr/lib/libaugeas.so.0: version `AUGEAS_0.16.0` not found (required by kdb)
+    /usr/lib/libaugeas.so.0: version `AUGEAS_0.16.0` not found (required by kdb)
 
-This is because ld tries to link /usr/lib/libaugeas.so.0 which is an older version of Augeas. Simply add
+This is because `ld` tries to link `/usr/lib/libaugeas.so.0` which is an older version of Augeas. Simply add
 the path to the newer library to your ld search paths (consult your system documentation on how to do this).
 
-
-## MOUNTING AND CONFIGURATION ##
+## Mounting and Configuration ##
 
 In order to mount the hosts file with the augeas plugin, issue the
 following command:
 
-	kdb mount /etc/hosts system/hosts augeas lens=Hosts.lns
+    kdb mount /etc/hosts system/hosts augeas lens=Hosts.lns
 
 The value of the plugin configuration option "lens" should be the
 module name of the lens (Hosts in the example) with a '.lns' suffix.
 Depending on your distribution and kind of installation, lenses can
-be found at `/usr/share/augeas/lenses/dist`,
+be found at `/usr/share/augeas/lenses/dist`, 
 `/usr/local/share/augeas/lenses/dist`, or something similar.
 The lens module name is equal to the filename without extension in pascal notation.
 For example, the lens `/usr/share/augeas/lenses/dist/hosts.aug` contains the module Hosts.
 
 Note that, without configuring the plugin to use a lens, the plugin
 will print an error message on the first usage:
 
-	kdb ls system/hosts
-	The command ls terminated unsuccessfully with the info: Error (#85) occurred!
-	Description: an Augeas error occurred
-	Ingroup: plugin
-	Module: storage
-	At: /path/augeas.c:166
-	Reason: Lens not found
+    kdb ls system/hosts
```

Comment:
> Btw. do you know something about the quality of the man pages remark generates?

No, but I can try to find out.

> Did you look how many warnings are issued by remark-lint?

With my special configuration that enforces quite a lot of stuff (e.g. same list character everywhere, equal indentations, atx-closed headers, blablabla...), it was about 4200 warnings for all markdown files together. After formatting with `remark`, the warnings increased to about 4900. So my desired still is quite apart from the remark default. 

Not sure if most/any of the settings are clear from their name, but that's what I used:
```
{
    "plugins": {
        "lint": {
            "blockquote-indentation": 2,
            "checkbox-character-style": {
                "checked": "x",
                "unchecked": " "
            },
            "checkbox-content-indent": true,
            "code-block-style": "indented",
            "definition-spacing": true,
            "fenced-code-marker": "`",
            "final-definition": true,
            "final-newline": true,
            "hard-break-spaces": true,
            "heading-increment": true,
            "heading-style": "atx-closed",
            "list-item-bullet-indent": true,
            "list-item-content-indent": true,
            "list-item-indent": "space",
            "list-item-spacing": true,
            "no-autolink-without-protocol": true,
            "no-blockquote-without-caret": true,
            "no-consecutive-blank-lines": true,
            "no-duplicate-definitions": true,
            "no-duplicate-headings": true,
            "no-emphasis-as-heading": true,
            "no-file-name-irregular-characters": true,
            "no-file-name-outer-dashes": true,
            "no-heading-content-indent": true,
            "no-heading-indent": true,
            "no-inline-padding": true,
            "no-literal-urls": true,
            "no-missing-blank-lines": {
                "exceptTightLists": true
            },
            "no-table-indentation": true,
            "no-tabs": true,
            "no-undefined-references": true,
            "table-cell-padding": "padded",
            "table-pipe-alignment": true,
            "table-pipes": true,
            "unordered-list-marker-style": "-"
        }
    }
}
```
If a setting is set to `true`, it normally means there is a check that warns if a situation described by the setting was found. E.g. `no-heading-indent` will warn if it finds something like `___## Blub` (underscore symbolizes a space) because of the leading space.